ow_power_system/src/power_system_node.cpp

```diff
@@ -102,6 +101,32 @@ void PowerSystemNode::powerCallback(const std_msgs::Float64::ConstPtr& msg)
   m_battery_temperature_pub.publish(battery_temperature_msg);
 }
 
+double PowerSystemNode::checkForFaults(double originalValue)
+{
+  //will likely need to modify after Chetan's feedback
+  // assuming that we can only have one power fault triggered at a time.
+  if (m_lowVoltageFault) {
+    std::cout << "low volt" << std::endl;
+    // return 30.0;
+  }
+  if (m_capLossFault) {
+    std::cout << "cap loss" << std::endl;
+    // return 20.0;
+  }
+  if (m_thermalFault) {
+    std::cout << "thermal" << std::endl;
```

Comment:
Assuming the couts are for debugging.   Use ROS_INFO, etc otherwise.

Reply:
I plan to fix this in a future PR when we know what values are needed